packages/calypso-e2e/src/lib/components/sidebar-component.ts

```diff
@@ -40,82 +42,43 @@ export class SidebarComponent {
 	 *
 	 * @param {string} item Plaintext representation of the top level heading.
 	 * @param {string} subitem Plaintext representation of the child level heading.
-	 * @param {number} retries Number of retries in the case of navigation failure.
 	 * @returns {Promise<void>} No return value.
 	 */
-	async navigate( item: string, subitem?: string, retries = 3 ): Promise< void > {
+	async navigate( item: string, subitem?: string ): Promise< void > {
+		await this.waitForSidebarInitialization();
+
 		if ( getTargetDeviceName() === 'mobile' ) {
 			await this.openMobileSidebar();
 		}
 
 		const itemSelector = `${ selectors.sidebar } :text-is("${ item }"):visible`;
-		await this.scrollItemIntoViewIfNeeded( itemSelector );
+		await this.page.dispatchEvent( itemSelector, 'click' );
```

Comment:
This is currently imbalanced in that we `waitForNavigation()` when using a subitem, but not when using a regular menu item. I think this is working because no top-level menu items trigger a true navigation to another page, but it might not be a bad idea to add that here as a safety net?

Reply:
I was originally going to say that if we add a `waitForNavigation` when a true navigation does not occur, it will likely trip up the execution because it will wait for a navigation that never occurs.

But I then looked back at the original code that Bart wrote, and it does have a `waitForNavigation()` call even only the top level menu is called.

I'll try adding the `waitForNavigation()` back in and run another stress test before merge. Thanks for the catch!

Reply:
Turns out, on mobile viewports a navigation isn't fired so having a `waitForNavigation()` ends up breaking this flow.

Inconsistency!

Reply:
Oh man, that inconsistency is brutal! It seems to work though in this current state? That might be the best we can do with the `waitForNavigation` differences between viewports.

Reply:
Yeah, I iterated a lot more to see if I can cover every single scenario. The stress tests came back 99% good, but there's always that one odd instance where the validation fails for some unknown reason.

It seems to be good for now, so I will merge it as-is. We can iterate more (sigh) if we need to.